zk/stages/stage_sequence_execute_injected_batch.go

```diff
@@ -31,7 +31,8 @@ func processInjectedInitialBatch(
 		return err
 	}
 
-	header, parentBlock, err := prepareHeader(batchContext.sdb.tx, 0, math.MaxUint64, math.MaxUint64, batchState.forkId, batchContext.cfg.zk.AddressSequencer)
+	coinbase := batchState.getCoinbase(batchContext.cfg)
```

Comment:
I would suggest to leave "injected batch" as it is by using constants rather than getting Coinbase from batchState, because it will always be equal to a constant so it is useless check.

Reply:
Recovered it